test/framework/src/main/java/org/elasticsearch/snapshots/AbstractSnapshotIntegTestCase.java

```diff
@@ -470,14 +470,24 @@ protected void createIndexWithRandomDocs(String indexName, int docCount) throws
     }
 
     protected void indexRandomDocs(String index, int numdocs) throws InterruptedException {
+        indexRandomDocs(index, numdocs, true);
```

Comment:
Are these simply test fixes related to this change?

Reply:
Yes, that's correct